src/main/java/org/cactoos/collection/Immutable.java

```diff
@@ -56,23 +55,6 @@
      */
     private final Collection<X> col;
 
-    /**
-     * Ctor.
-     * @param src Source elements
-     */
-    @SafeVarargs
-    public Immutable(final X... src) {
```

Comment:
@fanifieiev why should we remove this varargs ctor? If I'm right, we should get rid only of the `CollecitonOf`. Maybe, it's better to leave a todo for that refactoring?

Reply:
@iakunin Ok.

Reply:
@fanifieiev this comment was about ctor from Iterable, but not about varargs-ctor.

Reply:
@iakunin this is the same thing: this class is a decorator, it's not a way to create new collection, but to decorate them.

Reply:
@victornoel ok, got it. Let this ctor be removed.